components/admin-panel/Menu.js

```diff
@@ -44,11 +44,7 @@ const OrganizationSettingsMenuLinks = ({ collective, isAccountantOnly }) => {
       {!isAccountantOnly && (
         <React.Fragment>
           <MenuLink collective={collective} section={ORG_BUDGET_SECTIONS.TIERS} />
-          <MenuLink
-            collective={collective}
-            section={COLLECTIVE_SECTIONS.TIERS_REVAMP}
-            if={['development', 'staging'].includes(process.env.OC_ENV)}
```

Comment:
I think we should still wait a bit before actually releasing it and have it as a separate PR. Also, the new page should replace the previous one; there should be only one "Tiers" option in the menu.

We can eventually have an opt-out flag in case we want to quickly give access to the old settings for a given account.